test/requirements.txt

```diff
@@ -1,6 +1,7 @@
 pytest
 coverage
 torch
+torchaudio
```

Comment:
@MinaKh here's where you have to specify the CPU version of `torchaudio`. The CI pipeline installs the dependencies from this file.